mono/tests/test-runner.cs

```diff
@@ -163,20 +163,33 @@ class TestInfo {
 					test_info.Add (new TestInfo { test = s, opt_set = opt });
 			}
 		}		
+		var previous_test_hung = false;
```

Comment:
maybe rename this variable such that it's clear that it is only used for concurrency==1